components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer/src/main/java/org/wso2/carbon/deployment/synchronizer/repository/CarbonRepositoryUtils.java

```diff
@@ -92,29 +92,18 @@ public static DeploymentSynchronizer newCarbonRepositorySynchronizer(int tenantI
     public static DeploymentSynchronizerConfiguration getActiveSynchronizerConfiguration(
             int tenantId) throws DeploymentSynchronizerException {
 
-        try {
-            //Firrst attempt to get the configuration from carbon.xml
-            DeploymentSynchronizerConfiguration config = getDeploymentSyncConfigurationFromConf();
-
-            //If configuration has not been specified in carbon.xml
-            if (config == null) {
-                //Attempt to get configuration from local registry.
-                config = getDeploymentSyncConfigurationFromRegistry(tenantId);
-
-                //If configuration does not exist in the local registry as well.
-                if (config == null) {
-                    //Get default Deployment Synchronizer Configuration
-                    config = getDefaultDeploymentSyncConfiguration();
-                }
-            } else {
-                //If config is obtained from carbon.xml, set attribute to disable UI
-                config.setServerBasedConfiguration(true);
-            }
-            return config;
-        } catch (RegistryException e) {
-            throw new DeploymentSynchronizerException("Error while loading synchronizer " +
-                    "configuration from the registry", e);
+        //Firrst attempt to get the configuration from carbon.xml
+        DeploymentSynchronizerConfiguration config = getDeploymentSyncConfigurationFromConf();
+
+        //If configuration has not been specified in carbon.xml
+        if (config == null) {
+            //Get default Deployment Synchronizer Configuration
+            config = getDefaultDeploymentSyncConfiguration();
+        } else {
+            //If config is obtained from carbon.xml, set attribute to disable UI
+            config.setServerBasedConfiguration(true);
         }
+        return config;
```

Comment:
Add new line in between if block and return statement to increase readability.

Reply:
Will add a new line